test/files/run/t12290/TextBlocks.java

```diff
@@ -0,0 +1,68 @@
+class TextBlocks {
```

Comment:
This code only compiles if the Java compiler is new enough to support text blocks. Is there some way to indicate that?

Reply:
Unfortunately we don't have anything set up to test on newer JDKs than 8.

You could do something like https://github.com/scala/scala/blob/2.13.x/test/files/run/t8852a.scala, I think that was added in a similar situation when we were still on JDK 6.

But we should have something better here, since this will also come when adding support for records or sealed classes.